docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -0,0 +1,520 @@
+
+```
+
+2. Modify the domain resource YAML file to provide the secret names:
+
+```
+  configuration:
+    opss:
+      # Name of secret with walletPassword for extracting the wallet
+      walletPasswordSecret: sample-domain1-opss-wallet-password-secret      
+      # Name of secret with walletFile containing base64 encoded opss wallet
+      walletFileSecret: sample-domain1-opss-walletfile-secret
+
+```
+
+See [Reusing an RCU database]({{< relref "/userguide/managing-domains/model-in-image/reusing-rcu.md" >}}) for instructions.
+
+### Use the WebLogic Image Tool to create an image
+
+Model in Image must contain a WebLogic install, a WebLogic Deploy Tool install, and your WDT model files. You can use the sample `./build.sh` script to build this image, which will perform the following steps for you:
+
+  - Uses `docker pull` to obtain a base image. (See [Prerequisites for all domain types](#prerequisites-for-all-domain-types) to set up access to the base image.)
+  - Downloads the latest WebLogic Image Tool and WebLogic Deploy Tool to `WORKDIR`.
+  - Creates and populates a staging directory `$WORKDIR/models` that contains your WDT model files and WDT application archive.
+    - Builds  a simple servlet application in `$SAMPLEDIR/sample_app` into a WDT model application archive `$WORKDIR/models/archive1.zip`.
+    - Copies sample model files from `$SAMPLEDIR/model-in-image` to `$WORKDIR/models`. This uses a model file that is appropriate to the domain type (for example, the `JRF` domain model includes database access configuration).
+  - Uses the WebLogic Image Tool and the `$WORKDIR/models` staging directory to create a final image named `model-in-image:v1` that layers on the base image. Specifically, it runs the WebLogic Image Tool with its `update` option, which:
+    - Builds the final image as a layer on the base image.
+    - Puts a WDT install in image location, `/u01/wdt/weblogic-deploy`.
+    - Copies the WDT model, properties, and application archive from `$WORDKIR/models` to image location, `/u01/wdt/models`.
+
+The script expects `JAVA_HOME`, `WDT_DOMAIN_TYPE`, and `WORKDIR` to already be initialized. (See [Prerequisites for all domain types](#prerequisites-for-all-domain-types).)
+
+Run the script:
+
+  ```
+  $SAMPLEDIR/build.sh
+  ```
+
+If you intend to use a remote Docker registry, you need to tag and push the image to the remote Docker registry.
+
+1.  Tag the image for the remote Docker registry, for example:
+
+```
+docker tag <image-name>:<tag> <region-key>.ocir.io/<tenancy-namespace>/<repo-name>/<image-name>:<tag>
+```
+
+2.  Push the image to the remote Docker registry, for example:
+
+```
+docker push <region-key>.ocir.io/<tenancy-namespace>/<repo-name>/<image-name>:<tag>
+```
+
+3. Create the pull secret for the remote Docker registry:
+
+```
+ kubectl -n <domain namespace> create secret docker-registry <secret name> \
```

Comment:
The sample should not be based on OKE it should be based on a generic Kubernetes cluster similar to our other samples

Reply:
Tom AI: See above.

Reply:
Done.